Purchases/Public/Purchases.swift

```diff
@@ -1360,4 +1346,14 @@ private extension Purchases {
         }
     }
 
+    func purchase(product: SKProduct,
```

Comment:
this one is missing docs

Reply:
this is private, the public one is line 766